src/test/java/com/google/javascript/clutz/partial/missing_generic.js

```diff
@@ -0,0 +1,18 @@
+goog.module('missing.gen');
+
+//!! When emitting with missing sources, we have to emit T = any, to
+//!! support downsteam consumers that use missing.gen.C without passing
+//!! any type parameters.
+/**
+ * @constructor
+ * @template T
+ */
+function C(){}
```

Comment:
I'd really, really prefer to name these after the case they are testing. `C` says nothing at all, `GenericClassDefaultsArgumentsToAny` or even just `GenericClass` would make this so much more readable. You're not calling all the variables in your source code `a`, `b`, `c`, etc either.

I've been guilty of this (I think I introduced this pattern in the first place), but after debugging a bunch of test cases, I've really come around on it.